src/v2/coins/abstractUtxoCoin.js

```diff
@@ -133,11 +144,11 @@ class AbstractUtxoCoin extends BaseCoin {
   static inferAddressType(addressDetails) {
     if (_.isObject(addressDetails.coinSpecific)) {
       if (_.isString(addressDetails.coinSpecific.redeemScript) && _.isString(addressDetails.coinSpecific.witnessScript)) {
-        return this.AddressTypes.P2SH_P2WSH;
+        return Codes.UnspentTypeTcomb('p2shP2wsh');
```

Comment:
the only reason we should use this instead `Codes.UnspentType.p2shP2wsh` is that we get a runtime error on typo (except in `NODE_ENV=production`)

However with TypeScript we should get a compile-time error anyway, no?

Reply:
This is a js file. `Codes.UnspentType.pwshP2wsh` only exists for ts files.

Reply:
So actually this is now a ts file. However, it seems that `Codes.UnspentType` is not exported. I tried to hack around for a bit to get it exported but my typescript is poor and I couldn't figure it out. If we can export it then I agree that we should use it here.

Reply:
Put in these 2 tickets for this: https://bitgoinc.atlassian.net/browse/BG-11667 & https://bitgoinc.atlassian.net/browse/BG-11668